src/platform/site-wide/component-library-analytics-setup.js

```diff
@@ -464,6 +464,18 @@ const analyticsEvents = {
       action: 'change',
       event: 'int-radio-button-option-click',
       prefix: 'radio-button',
+      ga4: {
+        event: 'interaction',
+        component_name: 'va-radio',
+        custom_string_1: 'component_library',
+      },
+      /* Component to GA4 parameters */
+      mapping: {
+        'radio-button-label': 'custom_string_2',
+        'radio-button-optionLabel': 'value',
+        'radio-button-required': 'required',
```

Comment:
I don't think these are mapping correctly. Should the prefix just be `radio-label`, `radio-optionLabel`, `radio-required`?

```
'radio-button-label': 'custom_string_2',
'radio-button-optionLabel': 'value',
'radio-button-required': 'required',
```

![Screenshot 2023-07-10 at 2 48 37 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/872479/07701367-309f-4562-9ff1-704a7c07133b)

I would expect to see them in the console like:

```
custom_string_2: "something",
value: "something",
required: false
```

Reply:
good catch - fixed!